packages/expo-router/assets/native-tabs.module.css

```diff
@@ -0,0 +1,114 @@
+.nativeTabsContainer {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  max-height: 100vh;
+  max-width: 100vw;
+  overflow: hidden;
+}
+
+.tabContent {
+  flex: 1;
+  display: flex;
+  max-height: 100%;
+  max-width: 100%;
+}
+.tabContent[data-state="inactive"] {
+  display: none;
+}
+
+.navigationMenuRoot {
+  top: 24px;
+  left: 50%;
+  transform: translateX(-50%);
+  position: fixed;
+  z-index: 10;
+  display: flex;
+  background-color: var(--expo-router-native-tabs-background-color, #272727);
+  height: 40px;
+  border-radius: 25px;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 5px;
+  box-sizing: border-box;
+  margin: 0;
+  max-width: 90vw;
+  overflow-x: auto;
+}
+
+.navigationMenuTrigger {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  list-style-position: inside;
+  height: 100%;
+  background-color: transparent;
+  border: none;
+  margin: 0;
+  height: 100%;
+  border-radius: 20px;
+  padding: 0 20px;
+  cursor: pointer;
+  outline-color: #444444;
```

Comment:
Using css variables would potentially allow for overwriting colors with user-space css